packages/node_modules/@webex/plugin-meetings/src/share.js

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+const TYPES = ['application', 'screen', 'stream', 'whiteboard'];
```

Comment:
Is there a reason the `TYPES` and `STATUSES` are different? I prefer the object type instead of an array.

Also, probably would be helpful to export these consts so devs can use them.

Reply:
No good reason. It's changed.